spec_parser/utils.py

```diff
@@ -162,6 +163,123 @@ def gen_rdf(self) -> None:
         with safe_open(fname, "w") as f:
             f.write(g.serialize(format="turtle"))
 
+    def gen_schema(self) -> None:
```

Comment:
Do we include the description of the classes and fields in the schema?  If not, we may want to add those.

Reply:
The output is already 5000 lines long, so I decided to omit descriptions. If there are use-cases where they are actually needed, I can add them in, though.